tomatic/claims.py

```diff
@@ -10,6 +10,7 @@ class Resolution(str, Enum):
     fair = 'fair'
     unfair = 'unfair'
     irresolvable = 'irresolvable'
+    not_resolution = ''
```

Comment:
wow, the semantics were already fuzzy between unsolved and irresolvable, but not_resolution?? Why this?